src/__mocks__/clipboardUnit.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
```suggestion
export default {
```

Try to avoid `module.exports` if at all possible.

Reply:
Changed